### DISCUSSION TITLE
Sorry about that

### DISCUSSION THREAD

**Discussion by RetroDude128** [marceldobehere/MaslOS#16] [General]
My PC is acting up and I have no clue why, basically, It isn't working to the point where I can't even do anything on it, I'm using a computer at the library right now

I'll have to push the stream off to next week

**Comment by RetroDude128** [comment]
I really do hope you hop onto github and see this, this is the best thing I can do right now.

**Reply by Yo-yo-ooo** [reply]
What's Happing?
